src/main/java/org/sonar/plugins/stash/StashPlugin.java

```diff
@@ -41,7 +41,7 @@
   public static final String STASH_RESET_COMMENTS = "sonar.stash.comments.reset";
   public static final String STASH_URL = "sonar.stash.url";
   public static final String STASH_LOGIN = "sonar.stash.login";
-  public static final String STASH_PASSWORD = "sonar.stash.password";
+  public static final String STASH_PASS_PROPERTY = "sonar.stash.password";

```

Comment:
Regarding variables naming I don t see any issue to use "password" when the variable is a password.
The proper guideline is to be relevant in naming to have good readability of our code.